octodns/manager.py

```diff
@@ -520,6 +521,10 @@ def _populate_and_plan(
                     )
                     source.populate(zone)
 
+            # Frozen copy of zone records *before* going through processors so
+            # alias zone can copy them if needed.
+            zone._pre_processing_records = deepcopy(zone._records)
```

Comment:
Does `deepcopy` actually know how to make identical records? I would have thought it would be able to safely do that w/objects.

There is shallow copy support in Zone, but I'd have to page the details back into memory to know if it was useful here or not.